### PR TITLE
Specs refactor

### PR DESCRIPTION
**Pull request by lasalvavida**
First iteration of the test refactoring in #1259

### CODE REVIEW

File: Source/Scene/DebugAppearance.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ define([
         var getColor;
 
         // Well-known normalized vector attributes in VertexFormat
-        if ((attributeName === 'normal') || (attributeName === 'binormal') | (attributeName === 'tangent')) {
+        if ((attributeName === 'normal') || (attributeName === 'binormal') || (attributeName === 'tangent')) {
```
Comment (hpinkos) [inline Source/Scene/DebugAppearance.js]: :+1: good catch

File: Specs/Scene/GeometryRenderingSpec.js [hunk 2]
```diff
@@ -1548,25 +1512,30 @@ defineSuite([
...
+                                componentDatatype : ComponentDatatype.DOUBLE,
+                                componentsPerAttribute : 3,
+                                values : new Float64Array([
+                                    1000000.0, 0.0, 0.0,
+                                    1000000.0, 1000000.0, 0.0,
+                                    1000000.0, 0.0, 1000000.0,
+                                    1000000.0, 0.0, 1000000.0,
+                                    1000000.0, 1000000.0, 0.0,
+                                    1000000.0, 1000000.0, 1000000.0
+                                ])
```
Comment (lilleyse) [inline Specs/Scene/GeometryRenderingSpec.js]: Did you remove indices just for cleanup purposes?

Comment (lilleyse) [inline Specs/Scene/GeometryRenderingSpec.js]: I suppose it fits with the test below.

Comment (lasalvavida) [inline Specs/Scene/GeometryRenderingSpec.js]: Nope, that's a mistake, good catch

**Comment by lilleyse** [comment]
This all looks good to me. I just made some minor formatting tweaks.

**Pull request merged**